example/custom-models/ios/Runner/GoogleService-Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
you should not your GoogleService-Info.plist for security.

Reply:
Thanks, I thought I had deleted all my google files.